queries-lr.yaml

```diff
@@ -23,6 +23,11 @@ pg_stat_user_tables:
       autovacuum_count,
       analyze_count,
       autoanalyze_count
+      -- PostgreSQL 18+, cannot be added due backword compatibility: https://perconadev.atlassian.net/browse/PMM-14632
+      -- total_vacuum_time,
+      -- total_autovacuum_time,
+      -- total_analyze_time,
+      -- total_autoanalyze_time,
```

Comment:
It turned out that CASE statement in SQL cannot properly handle PG versions.
Disabled for time being: https://perconadev.atlassian.net/browse/PMM-14632